api/src/main/java/org/eclipse/microprofile/reactive/messaging/package-info.java

```diff
@@ -53,5 +53,5 @@
  * </pre>
  */
 
-@org.osgi.annotation.versioning.Version("1.1")
+@org.osgi.annotation.versioning.Version("3.0")
```

Comment:
The new version should be 2.0 not 3.0. Every package version should tell whether there are changes in a release. This version is not the version of spec.